.travis.yml

```diff
@@ -118,3 +123,6 @@ matrix:
     - env: BUILD_TARGET="posix-ncp-spi" VERBOSE=1
       os: linux
       compiler: gcc
+    - env: BUILD_TARGET="toranj-test-framework" VERBOSE=1 VIRTUAL_TIME=1
```

Comment:
The `VIRTUAL_TIME` env var only applies to `examples/Makefile-posix` and [tests/scripts/thread-cert](https://github.com/openthread/openthread/tree/master/tests/scripts/thread-cert).

Reply:
Will remove this from this commit/PR... Later we can enable virtual time more (with more testing)...